Dockerfile

```diff
@@ -315,13 +315,32 @@ RUN --mount=type=bind,from=auto-gptq-cache,src=/usr/src/auto-gptq-wheel,target=/
     pip install /usr/src/auto-gptq-wheel/*.whl --no-cache-dir
 
 # Install server
+# git is required to pull the fms-extras dependency
+RUN dnf install -y git && dnf clean all
 COPY proto proto
 COPY server server
 RUN cd server && make gen-server && pip install ".[accelerate, ibm-fms, onnx-gpu, quantize]" --no-cache-dir
 
 # Patch codegen model changes into transformers 4.35
 RUN cp server/transformers_patch/modeling_codegen.py ${SITE_PACKAGES}/transformers/models/codegen/modeling_codegen.py
 
+
+## Final Inference Server image ################################################
+FROM cuda-runtime as server-release
+ARG PYTHON_VERSION
+ARG SITE_PACKAGES=/opt/tgis/lib/python${PYTHON_VERSION}/site-packages
+
+# Install C++ compiler (required at runtime when PT2_COMPILE is enabled)
+RUN dnf install -y gcc-c++ && dnf clean all \
```

Comment:
I removed the `git` install from here since we now no longer need it to do pip installs in the final release stage. But we could put it back if we think it's useful to have on hand